.github/ISSUE_TEMPLATE/new_dataset.md

```diff
@@ -0,0 +1,21 @@
+---
+name: New dataset
+about: Provide information about a new dataset you'd like to see in PUDL
+title: ''
+labels: new-dataset
+assignees: ''
+---
+
+### Overview
+
+What is this dataset? Why do you want it in PUDL? Is it already partially in
+PUDL, or do we need to start from scratch?
```

Comment:
"Include any relevant information about the frequency of data update and licensing restrictions" might be a useful prompt here.